### DISCUSSION TITLE
How to use submission_information and make it run.

### DISCUSSION THREAD

**Discussion by nayyargaurav-aristocrat** [fastlane/fastlane#18436] [Q&A]
I am using deliver for upload to apple connect while doing that I am using submission_information to resolve the below issue.

![image](https://user-images.githubusercontent.com/60127493/111978599-190b7280-8b2a-11eb-88d9-931315fe21f1.png)

But even given the parameter in deliver shown below. I am still facing the same as above.

![image](https://user-images.githubusercontent.com/60127493/111978739-448e5d00-8b2a-11eb-825e-9bfafe0807ce.png)

Now I am confused about this just need to know if I missing something. or doing wrong and how I can resolve the same?

**Comment by nayyargaurav-aristocrat** [comment]
@Dinsen @max-ott @MartinGonzalez @marumemomo  @milch @tmm1 @Lutzifer @KrauseFx @minuscorp 
 anything you guys can suggest?

**Reply by minuscorp** [reply]
I don't think this is a proper way to ask for help.

**Reply by nayyargaurav-aristocrat** [reply]
@minuscorp Really sorry if you felt that. can you please guide me where I can ask questions and it will reach the collaborators?

Thanks in advance :)

**Reply by minuscorp** [reply]
We are all collaborators, so nobody gets paid for attending requests, we do it for free because we try to make a project even better than it is. The manner to get a collaborator attention is to open an issue or a request and wait to get attended.

Pinging a dozen of collaborators on purpose I think is mean against our spare time.

**Comment by joshdholtz** [comment]
@nayyargaurav-aristocrat Do you have your `deliver` option’s configured to set `submit_for_review: true`? The `submission_information` only gets used if `sumit_for_review` is set to `true`. I can’t tell by the information that you provided.

But... I would suggest setting `ITSAppUsesNonExemptEncryption` to `TRUE` in your Info.plist so that _fastlane_ doesn’t need to do it.

And if you could refrain from mass mentioning people to get your discussion seen that would be greatly appreciated 😊 As mentioned by other people, the core contributors aren’t paid and do this in their free time. We like to get this community as positive, friendly, and fun while remaining helpful. Spam mentioning contributors _could_ might not get the result that you’d like. Some of those people you mentioned aren’t even working on the project anymore 😉 

If you _do_ need to mention someone, please mention me as am the lead maintainer (I was not mentioned above 😓) but please don’t abuse that either as there are many aspects of _fastlane_ that need my attention 😇
